src/BuiltinExtensions/ComfyUIBackend/WorkflowGeneratorSteps.cs

```diff
@@ -333,7 +333,14 @@ public static void Register()
         #region Positive Prompt
         AddStep(g =>
         {
-            g.FinalPrompt = g.CreateConditioning(g.UserInput.Get(T2IParamTypes.Prompt), g.FinalClip, g.UserInput.Get(T2IParamTypes.Model), true, "6");
+            if (g.UserInput.TryGet(T2IParamTypes.EditPrompt, out string editPrompt) && !string.IsNullOrWhiteSpace(editPrompt))
```

Comment:
this if block shouldn't be duplicating the entire code inside, it's only changing one variable